package.json

```diff
@@ -22,7 +22,8 @@
     "test": "jest",
     "build:es5": "./node_modules/.bin/babel src -d lib",
     "build:webpack": "./node_modules/.bin/webpack lib/register.js",
-    "build": "yarn build:es5 && yarn test && yarn build:webpack"
+    "build": "yarn build:es5 && yarn test && yarn build:webpack",
```

Comment:
Is there some reason the es5 build happens before test?  Seems like it would waste some time transpiling in situations where the tests fail anyway.

Reply:
Yes they test the es5 version as well